nuntium/views.py

```diff
@@ -133,6 +132,7 @@ def get_context_data(self, form, **kwargs):
             context['persons'] = self.writeitinstance.persons.all()
         if self.steps.current == 'preview':
             context['message'] = self.get_form_values()
+            context['message']['people'] = context['message']['persons']
```

Comment:
I think that that the problem here is unify the naming related to ```Message.persons``` and ```Message.people``` for which I'm opening a new ticket.